misc/de-metas-common/de-metas-common-bpartner/src/main/java/de/metas/common/bpartner/v2/response/JsonResponseBPBankAccount.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * #%L
+ * de-metas-common-bpartner
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.common.bpartner.v2.response;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import de.metas.common.changelog.JsonChangeInfo;
+import de.metas.common.rest_api.common.JsonMetasfreshId;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+import javax.annotation.Nullable;
+
+@Value
+public class JsonResponseBPBankAccount
```

Comment:
consider using `@Jacksonized`

Reply:
in fact I want the good old JsonProperty @ to link the field names like `BPARTNER_ID`